docs/content/integrations/embedded-elt/dlt.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: "Dagster Embedded ELT: dlt"
+description: Ingest data with ease using Dagster and dlt
+---
+
+## dlt
+
+To get started with `dagster-embedded-elt` and _dlt_, first familiarize yourself with the quick [introduction to dlt](https://dlthub.com/docs/intro). When using the _dlt_ integration, you can either write your own sources, or use one of the many pre-made [verified sources](https://dlthub.com/docs/dlt-ecosystem/verified-sources/) in the _dlt_ ecosystem.
+
+## 1. Configuring your Dagster project to support `dlt`
+
+You will need to define a location for the `dlt` code used for ingesting data. We recommend creating a `dlt_sources` directory at the root of your Dagster project, however, this code can reside anywhere within your Python project.
+
+```bash
+cd $DAGSTER_HOME && mkdir dlt_sources
+```
+
+## 2. Initialize `dlt` ingestion code
+
+In your `dlt_sources/` directory, you can write your own ingestion code following the [dlt tutorial](https://dlthub.com/docs/tutorial/load-data-from-an-api), or you can opt to use a verified source. Here, we are going to demonstrate how you integrate the [GitHub source](https://dlthub.com/docs/dlt-ecosystem/verified-sources/github) provided by `dlt`.
+
+```bash
+$ cd dlt_sources/
+
+$ dlt init github snowflake
+Looking up the init scripts in https://github.com/dlt-hub/verified-sources.git...
+Cloning and configuring a verified source github (Source that load github issues, pull requests and reactions for a specific repository via customizable graphql query. Loads events incrementally.)
+Do you want to proceed? [Y/n]: y
+
+Verified source github was added to your project!
+* See the usage examples and code snippets to copy from github_pipeline.py
+* Add credentials for snowflake and other secrets in ./.dlt/secrets.toml
+* requirements.txt was created. Install it with:
+pip3 install -r requirements.txt
+* Read https://dlthub.com/docs/walkthroughs/create-a-pipeline for more information
+```
+
+This has pulled down the code required to easily collect data from the GitHub API. It has also created a `requirements.txt` and a `.dlt/` configuration. In this case, we can remove these files, as we will be handling configuration with environment variables, and the requirements are to be defined in Dagster's dependencies specification.
+
+```bash
```

Comment:
wonder if it's better to just do the `rm` command rather than show the files?

Reply:
I think it may be beneficial to keep it, as you can reference the requirements.txt and `.dlt/secrets.yaml` to understand what needs to be added elsewhere in the project.